Memories/Memories/VCs/AddViewController.swift

```diff
@@ -0,0 +1,159 @@
+//
+//  AddViewController.swift
+//  Test
+//
+//  Created by Alexander Supe on 13.03.20.
+//  Copyright © 2020 Alexander Supe. All rights reserved.
+//
+
+import UIKit
+import MapKit
+import CoreImage.CIFilterBuiltins
+
+class AddViewController: UIViewController, CLLocationManagerDelegate {
+    
+    @IBOutlet weak var imageView: UIImageView!
+    @IBOutlet weak var titleField: UITextField!
+    @IBOutlet weak var imageButton: UIButton!
+    @IBOutlet weak var filterButton: UIButton!
+    
+    @IBOutlet weak var audioButton: UIButton!
+    @IBOutlet weak var videoButton: UIButton!
+    @IBOutlet weak var audioView: UIView!
+    @IBOutlet weak var videoView: UIView!
+    
+    var titleText: String?
+    var image: Data?
+    var originalImage: Data?
+    var audio: String? { UserDefaults.standard.string(forKey: "CurrentAudio") }
+    var video: String? { UserDefaults.standard.string(forKey: "CurrentVideo") }
+    
+    var picker = UIImagePickerController()
+    let locationManager = CLLocationManager()
+    var coordinates: CLLocationCoordinate2D? { didSet { finishSaving() }}
+    private let context = CIContext(options: nil)
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        picker.delegate = self
+        UserDefaults.standard.removeObject(forKey: "CurrentAudio")
+        UserDefaults.standard.removeObject(forKey: "CurrentVideo")
+    }
+    
+    @IBAction func textFieldReturn(_ sender: UITextField) {
+        sender.resignFirstResponder()
+    }
+    
+    @IBAction func addImage(_ sender: Any) {
+        picker.allowsEditing = true
+        let actionSheet = UIAlertController(title: "Photo Source", message: "Choose a source", preferredStyle: .actionSheet)
+        
+        actionSheet.addAction(UIAlertAction(title: "Camera", style: .default , handler: { (sction: UIAlertAction) in
+            if UIImagePickerController.isSourceTypeAvailable(.camera) {
+                self.picker.sourceType = .camera
+                self.present(self.picker, animated: true, completion: nil) }
+            else { print("Camera not available") } }))
+        actionSheet.addAction(UIAlertAction(title: "Photo Libary", style: .default , handler: { (sction: UIAlertAction) in
+            self.picker.sourceType = .photoLibrary
+            self.present(self.picker, animated: true, completion: nil) }))
```

Comment:
The brackets and the if/else get lost because you're squashing if else statements into single lines.

These should be spaced out, especially since this logic is dense when compressed. It's hard to read.

```
actionSheet.addAction(UIAlertAction(title: "Camera", style: .default , handler: { (sction: UIAlertAction) in
    if UIImagePickerController.isSourceTypeAvailable(.camera) {
        self.picker.sourceType = .camera
        self.present(self.picker, animated: true, completion: nil)
    } else { print("Camera not available")
        
    }
}))
```